duckduckgo/duckduckgo-privacy-extension
# Implement browserify

**Reviewer:** @dharb 



## Description:

**Note** this doesn't merge into develop

This actually switches over the background build system to use browserify. There's still a few warts here and there. I'm going to add loads upon loads of comments where I can.

This diff looks much nicer when you ignore whitespace: https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/andrey/background-browserify...andrey/actually-browserify?w=1

## Steps to test this PR:

1. Build the extension, there should be no errors
2. Try using the extension normally, everything should look fine
3. There should be no console errors for the background page as you use the extension

Sorry I can't give you more detailed steps to test. I'm not sure if it's worth trying to catch out individual edge cases, since we're gonna integration test before `andrey/background-browserify` is merged into `develop` anyway.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
